detox/src/devices/drivers/android/tools/AppInstallHelper.js

```diff
@@ -1,35 +1,26 @@
-const EMU_TEMP_PATH = '/data/local/tmp';
-const EMU_TEMP_INSTALL_PATH = `${EMU_TEMP_PATH}/detox`;
-
 class AppInstallHelper {
```

Comment:
Extracted the contained logic of transferring files onto the device to a dedicated class (`FileXfer`)